src/VirtualClient/VirtualClient.Actions/SuperBenchmark/SuperBenchmarkExecutor.cs

```diff
@@ -82,6 +85,17 @@ public string ConfigurationFile
             }
         }
 
+        /// <summary>
+        /// Link to the superbench repo.
+        /// </summary>
+        public string RepositoryLink
```

Comment:
Do we need to change the repository link? Even if we need to support extra config?